test/library/characterization/test_readout_angle.py

```diff
@@ -58,6 +58,9 @@ def test_kerneled_expdata_serialization(self):
 
         exp = ReadoutAngle([0])
 
+        # Checking serialization of the experiment obj
+        self.assertRoundTripSerializable(exp)
```

Comment:
```suggestion
        self.assertRoundTripSerializable(exp.circuits())
```

Reply:
Edited as suggested. I had this test in another function but its redundant.